### ISSUE TITLE
stating the files blocks

### ISSUE THREAD

**Issue by Witiko** [mafintosh/torrent-mount#20]
Hey there,

when I `torrent-mount` a torrent file or a magnet link into my fs, any `stat(2)` on the mounted files blocks indefinitely meaning I can't do even a `ls -l` without my term freezing.

```
$ uname -sr
Linux 3.11.0-20-generic
$ npm --version
1.3.10
$ node --version
v0.10.25
```

**Comment by Munter** [comment]
Sounds like a missing or wrong callback in the `getattr` function. I had the same thing when I made a mistake implementing it. Probably related to the switch to the new fuse-bindings instead of fuse4js

**Comment by mafintosh** [comment]
@witiko latest version of torrent-mount?

**Comment by Witiko** [comment]
@mafintosh Yep.

**Comment by mafintosh** [comment]
Fixed in 1.4.1 - it was using some mount options that were unsupported by linux

**Comment by Witiko** [comment]
Confirming the fix. I love this.
